src/logic/addressBook/store/actions/addOrUpdateAddressBookEntry.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import { createAction } from 'redux-actions'
+
+import type { AddressBookEntryType } from '~/logic/addressBook/model/addressBook'
+
+export const ADD_OR_UPDATE_ENTRY = 'ADD_OR_UPDATE_ENTRY'
```

Comment:
given that we have this action, do we need `add_entry` one at all?

Reply:
I didn't wanted to mess the notifications.

Reply:
Makes sense. Maybe we can make action more specific like `UPDATE_ENTRY`? Or did you run into some problem where you tried to update an entry but it didn't exist?

Reply:
Actually I ran into a problem when tried to `ADD_ENTRY` and it already existed and, as both reducers (`ADD_ENTRY`, `UPDATE_ENTRY`) have their conditions, so I decided to go with a new one for this particular scenario.

There's, anyway, more to do around this OwnerList/AddressBook sync. I was about to point my concern here, but then I found this #588 (and there are several other bugs related to OwnerList).

My decision: let everything else works as it was working until now. I'll fix this particular bug, and maybe set some basis for future improvements. But let's make it incrementally so we don't introduce new bugs.